omnibus/files/private-chef-cookbooks/private-chef/templates/default/opc_logrotate.cron

```diff
@@ -3,4 +3,4 @@
 PATH=/opt/opscode/embedded/sbin:$PATH
 
 command -v logrotate >/dev/null 2>&1 || exit 0
-nice -n 19 ionice -c3 logrotate /etc/opscode/logrotate.conf
+nice -n 19 ionice -c3 logrotate -s /var/log/chef/logrotate.status /etc/opscode/logrotate.conf
```

Comment:
Overall this looks good, my only question is whether `/var/log/chef` is the most appropriate path for this.